### PR TITLE
Some optimizations to crawling

### PR DESCRIPTION
**Pull request by meriouma**

### CODE REVIEW

**Comment by olafleur** [comment]
LGTM

File: src/main/webapp/WEB-INF/appengine-web.xml [hunk 1]
```diff
@@ -21,4 +21,7 @@
         <property name="java.util.logging.config.file" value="WEB-INF/logging.properties"/>
     </system-properties>
 
+    <automatic-scaling>
+        <min-idle-instances>1</min-idle-instances>
```
Comment (jasonlemay) [inline src/main/webapp/WEB-INF/appengine-web.xml]: No extra cost to have one instance always on?

**Comment by jasonlemay** [comment]
LGTM

File: src/main/webapp/WEB-INF/appengine-web.xml (hunk 1 above)
Comment (meriouma) [inline src/main/webapp/WEB-INF/appengine-web.xml]: It's our website anyway, we don't want to have cold starts. And if the traffic is low enough, it's still free (the first 28h are free)

**Comment by christiangoudreau** [comment]
Please, pay attention to run the right kind of instance if we don't want
any extra cost. Last time we made that error and paid for it :D

**Comment by meriouma** [comment]
Last time we weren't using automatic scaling. I only activated a resident instance

**Pull request merged**